builtin/providers/aws/resource_aws_load_balancer_policy.go

```diff
@@ -0,0 +1,239 @@
+		dlbOpts := &elb.DescribeLoadBalancersInput{
+			LoadBalancerNames: []*string{lbspOpts.LoadBalancerName},
+		}
+
+		lbDesc, err := elbconn.DescribeLoadBalancers(dlbOpts)
+		if err != nil {
+			return fmt.Errorf("Error retrieving ELB: %s", err)
+		}
+
+		policyNames := []*string{}
+
+		for _, backendServerDesc := range lbDesc.LoadBalancerDescriptions[0].BackendServerDescriptions {
+			foundInstancePort := backendServerDesc.InstancePort
+			if instancePort == *foundInstancePort {
+				policyNames = append(backendServerDesc.PolicyNames, lbspOpts.PolicyName)
+			}
+		}
+
+		if len(policyNames) == 0 {
+			policyNames = append(policyNames, lbspOpts.PolicyName)
+		}
+
+		lbpbsOpts := &elb.SetLoadBalancerPoliciesForBackendServerInput{
+			InstancePort:     aws.Int64(int64(instancePort)),
+			LoadBalancerName: lbspOpts.LoadBalancerName,
+			PolicyNames:      policyNames,
+		}
+
+		if _, err := elbconn.SetLoadBalancerPoliciesForBackendServer(lbpbsOpts); err != nil {
+			return fmt.Errorf("Error setting BackendServerAuthenticationPolicyType: %s", err)
+		}
+	}
+
+	d.SetId(fmt.Sprintf("%s:%d:%s",
+		*lbspOpts.LoadBalancerName,
+		instancePort,
+		*lbspOpts.PolicyName))
+	return nil
+}
+
+func resourceAwsLoadBalancerPolicyRead(d *schema.ResourceData, meta interface{}) error {
+	elbconn := meta.(*AWSClient).elbconn
+
+	lbName, _, policyName := resourceAwsLoadBalancerPolicyParseId(d.Id())
+
+	request := &elb.DescribeLoadBalancerPoliciesInput{
+		LoadBalancerName: aws.String(lbName),
+		PolicyNames:      []*string{aws.String(policyName)},
+	}
+
+	getResp, err := elbconn.DescribeLoadBalancerPolicies(request)
+	if err != nil {
+		if ec2err, ok := err.(awserr.Error); ok && ec2err.Code() == "PolicyNotFound" {
+			// The policy is gone.
+			d.SetId("")
+			return nil
+		}
+		return fmt.Errorf("Error retrieving policy: %s", err)
+	}
+
+	if len(getResp.PolicyDescriptions) != 1 {
+		return fmt.Errorf("Unable to find policy %#v", getResp.PolicyDescriptions)
+	}
+
+	policyDesc := getResp.PolicyDescriptions[0]
+	policyTypeName := policyDesc.PolicyTypeName
+
+	d.Set("policy_name", policyName)
+	d.Set("policy_type_name", policyTypeName)
+	d.Set("load_balancer_name", lbName)
+
+	return nil
+}
+
+func resourceAwsLoadBalancerPolicyDelete(d *schema.ResourceData, meta interface{}) error {
+	elbconn := meta.(*AWSClient).elbconn
+
+	lbName, _, policyName := resourceAwsLoadBalancerPolicyParseId(d.Id())
+
+	instancePortInt := d.Get("instance_port")
+	if instancePortInt == 0 {
+		instancePortInt = 1
+	}
+
+	policyTypeName := d.Get("policy_type_name")
+
+	if policyTypeName == "BackendServerAuthenticationPolicyType" {
+		policyNames := []*string{}
+
+		lbpbsOpts := &elb.SetLoadBalancerPoliciesForBackendServerInput{
+			InstancePort:     aws.Int64(int64(instancePortInt.(int))),
+			LoadBalancerName: aws.String(lbName),
+			PolicyNames:      policyNames,
+		}
+
+		if _, err := elbconn.SetLoadBalancerPoliciesForBackendServer(lbpbsOpts); err != nil {
```

Comment:
in order to *delete* the policies... the AWS sdk requires setting the policies to an empty data structure. see line 209 above.